app/models/work_group.rb

```diff
@@ -4,6 +4,7 @@
 #
 #  id              :bigint           not null, primary key
 #  name            :string
+#  time_limit      :integer
```

Comment:
would be good to add a comment of what it is - minutes? seconds?

Reply:
also needs to add rotation time

Reply:
okay understood, will get it fixed 👍🏻